Document/CategoryManager.php

```diff
@@ -22,7 +22,7 @@ class CategoryManager extends BaseDocumentManager implements CategoryManagerInte
     /**
      * {@inheritdoc}
      */
-    public function getPager(array $criteria, $page, $maxPerPage = 10)
+    public function getPager(array $criteria, $page, $limit = 10, array $sort = array())
```

Comment:
@greg0ire I don't know if it should be a major or a patch for this case because this class already inherit from core-bundle class that have this signature since the begining...

ping @rande.